internal/plugin_config_test.go

```diff
@@ -28,6 +28,28 @@ data:
 	}
 }
 
+func createIamPolicyTypeConfigMap(t *testing.T, tmpDir, filename string) {
```

Comment:
This is clearer in my opinion.
```suggestion
func createIamPolicyManifest(t *testing.T, tmpDir, filename string) {
```